grouper/group.py

```diff
@@ -20,6 +21,36 @@ def get_all_groups(session):
     return session.query(Group).filter(Group.enabled == True)
 
 
+def get_audited_groups(session):
+    # type: (Session) -> List[Group]
+    """Returns all audited enabled groups.
+
+    At present, this is not cached at all and returns the full list of
+    groups from the database each time it's called.
+
+    Args:
+        session (Session): Session to load data on.
+
+    Returns:
+        a list of all enabled and audited Group objects in the database
+    """
+    audited_groups = []
+    graph = Graph()
+    for group in get_all_groups(session):
+        try:
+            group_md = graph.get_group_details(group.name)
+        except NoSuchGroup:
+            # Very new group with no metadata yet, or it has been disabled and
+            # excluded from in-memory cache.
+            group_md = {}
```

Comment:
We can just `continue` here instead, since all the rest of this loop is going to be a noop